apps/base-docs/docs/pages/identity/smart-wallet/guides/paymasters.mdx

```diff
@@ -1,52 +1,75 @@
+import { Callout } from "vocs/components"
+
 # Paymasters (Sponsored Transactions)
 
 One of the biggest UX enhancements unlocked by Smart Wallet is the ability for app developers to sponsor their users' transactions. If your app supports Smart Wallet, you can start sponsoring your users' transactions by using [standardized paymaster service communication](https://erc7677.xyz) enabled by [new wallet RPC methods](https://eip5792.xyz).
 
 The code below is also in our [Wagmi Smart Wallet template](https://github.com/wilsoncusack/wagmi-scw/).
 
-## Using Wagmi + Permissionless in a Next.js app
```

Comment:
How are we thinking about this tutorial in relation to https://docs.base.org/use-cases/go-gasless ?


Could we further contextualize at the start here that this is a companion tutorial for a starter project, instead of the holistic docs?